evergreen/config.yml

```diff
@@ -431,6 +431,7 @@ functions:
         if ls crash-*> /dev/null 2>&1; then
           echo "Found crash file"
           mv crash-* realm-fuzzer-crash.txt
+          mv fuzz-test.realm fuzzer_realm.realm
```

Comment:
In theory, the realm file we use for the fuzzer is never deleted, so it is always reused (cycle after cycle). If I got a crash file, I know there is a crash, so I am uploading the crash report and the realm, in order to avoid uploading for every cycle the realm file, I am just renaming it. 
I am not sure, if there is a better strategy. So essentially, I only need to upload the file if there is a failure.